backend/entrypoint.sh

```diff
@@ -14,6 +14,16 @@ echo "Running backend server..."
 
 python manage.py rqworker default 2>&1 | tee /logs/rqworker.log &
 
+echo "Installing crontab..."
+
+if [ -z "$IMAGE_SCAN_SCHEDULE" ]
+then
+    IMAGE_SCAN_SCHEDULE="0 */6 * * *"
+fi
+
+echo "$IMAGE_SCAN_SCHEDULE python3 manage.py scan >/dev/null 2>&1" > crontab
```

Comment:
Why hide errors with `>/dev/null 2>&1` here?

Reply:
No need to redirect stdout to /dev/null here, removed

Reply:
My bad, I misunderstood the point of that. So it will write `0 */6 * * * python3 manage.py scan >/dev/null 2>&1` and it should prevent the cron task from crashing if there's an exception when running the Python script.

Reply:
Yeah that's right, it's building the crontab. 

Idea of `2>&1` is to redirect stderr to stdout so we can see it in the cron logs (which may not always write stderr).

The redirection to `/dev/null` was not a good idea because it would discard all the output, which may be useful for troubleshooting